src/BloomBrowserUI/ePUB/baseEPUB.less

```diff
@@ -61,6 +61,9 @@ div.bloom-editable {
         margin-top: auto;
     }
     img.branding {
+        // allow for more than one image
+        display: flow;
```

Comment:
This is still an experimental display mode... I've never even heard of it. It may well not be supported by readers. Is there a way to accomplish this with something more standard?

Reply:
OK, I'll try for another PR with a different display mode that is more standard but does the job.